test/kubernetes-test.sh

```diff
@@ -102,4 +102,4 @@ ${HOME}/go/bin/kubetest --provider=local --test \
 # should enforce ingress policy allowing any port traffic to a server on a specific protocol
```

Comment:
Please add - 
"Cilium does not support internalTrafficPolicy as Local Redirect Policy allows user to enable node-local redirection."